src/clj/puppetlabs/services/config/puppet_server_config_service.clj

```diff
@@ -27,11 +27,11 @@
           (ks/pprint-to-string puppet-config))
         (core/init-webserver! override-webserver-settings!
                               (get-in tk-config
-                                      [:webserver :puppet-server]
+                                      [:webserver :puppetserver]
```

Comment:
I put the dash back in for this in 2a4d62e.  Assuming we think this PR can be merged before @cprice404 but decide later on that we still really do want to change it, maybe we could do that in a follow-up PR - and submit separate tickets to track the corresponding pe-puppetserver config and PE module config changes that would presumably need to go along with it.